runtime/oti/j9nonbuilder.h

```diff
@@ -5034,6 +5034,9 @@ typedef struct J9VMThread {
 #endif /* OMR_GC_COMPRESSED_POINTERS */
 #endif /* OMR_GC_CONCURRENT_SCAVENGER */
 	UDATA safePointCount;
+#if defined(J9VM_OPT_SNAPSHOTS)
+	UDATA *restoreThreadBP;
```

Comment:
Can you add a comment describing to the code describing the purpose of this field?